pkg/identity/identitysync/okta.go

```diff
@@ -50,22 +55,50 @@ func (s *OktaSync) TestConfig(ctx context.Context) error {
 
 // userFromOktaUser converts a Okta user to the identityprovider interface user type
 func (o *OktaSync) idpUserFromOktaUser(ctx context.Context, oktaUser *okta.User) (identity.IDPUser, error) {
+	log := logger.Get(ctx).With("okta.user_id", oktaUser.Id)
+
+	userJSON, err := json.Marshal(oktaUser)
+	if err != nil {
+		log.Errorw("error marshalling user for logging", zap.Error(err))
+	}
+
+	log.Debugw("converting okta user to internal user", "oktaUser", string(userJSON))
+
+	firstName, ok := (*oktaUser.Profile)["firstName"].(string)
+	if !ok {
+		log.Error("okta profile had no firstName")
```

Comment:
If these are intended to be non blocking errors, we should add a comment in the code so we know why

Reply:
Thanks, I've added a comment here